lib/shotstack/models/output.rb

```diff
@@ -167,7 +167,7 @@ def list_invalid_properties
     # @return true if the model is valid
     def valid?
       return false if @format.nil?
-      format_validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3"])
+      format_validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3", "jpg"])
```

Comment:
Please can you add to line 187 as well `validator = EnumAttributeValidator.new('String', ["mp4", "gif", "mp3"])`

Reply:
Let me merge the changes and I'll add this in. I will need to update the version as well and deploy to rubygems.